src/content/docs/extensions/algos/index.mdx

```diff
@@ -0,0 +1,219 @@
+---
+title: Graph algorithms
+---
+
+The graph algorithms extension package allows you to directly run popular graph algorithms
+like PageRank, connected components and Louvain on your graph data stored in Kuzu. Using this extension,
+you do not need to export your data to specialized graph analytics tools like NetworkX (at
+least for the algorithms that are supported by the extension). The algorithms run natively
+in Kuzu, which also allows you to scale to very large graphs!
+
+Graph algorithms are useful tools for extracting meaningful insights from connected data.
+Whether you're detecting fraud patterns in financial transactions, optimizing supply chain networks,
+or analyzing social media interactions, these algorithms help you understand complex relationships and
+make data-driven decisions. The following sections describe how to use the graph algorithms extension
+in Kuzu.
+
+## Project graph
```

Comment:
I suggest moving "Project graph" part to a separate page. In this "Index" page, we should focus more on listing supported algorithms, which is not available now.

Reply:
I moved it to this main page because the concept of projecting graphs is critical to using graph algos. And I want to avoid multiple levels of depth in the docs page as far as possible (unless there's no other way). I suggest we keep this in the main page and let people read about projections in the overview page itself for now.

Reply:
I see. Then let's keep "Project graph" here as it is.